buildtest/cli/__init__.py

```diff
@@ -658,7 +658,7 @@ def config_menu(subparsers):
 
     compilers = subparsers_config.add_parser("compilers", help="Search compilers")
 
-    subparsers_config.add_parser("edit", help="Open configuration file in editor")
+    subparsers_config.add_parser("edit-test", help="Open configuration file in editor")
```

Comment:
So this method `config_menu` is for building the arguments for `buildtest config` and we want to keep `buildtest config edit` command so please revert this change.